.github/workflows/test.yml

```diff
@@ -11,20 +11,14 @@ jobs:
   jest:
     name: Unit Tests
     runs-on: ubuntu-latest
-    env:
-      CI: true
```

Comment:
CI: true is not required anymore

https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/